src/workflow/workflowOptions.ts

```diff
@@ -74,8 +75,46 @@ export const workflowOptions = {
     ],
     initial: 0,
   },
+  useSurf: {
+    type: (prev, values) => needSurfChoice(values),
+    name: "useSurf",
+    message:
+      "Choose if you want to use Surf, the TypeScript type safety tool maintained by Thala Labs",
```

Comment:
could we add a link to https://aptos.dev/en/build/sdks/ts-sdk/type-safe-contract
```suggestion
      "Would you like to use Surf, the TypeScript type safety tool maintained by Thala Labs",
```